policies/bundles/WebApplicationFirewallEnabled/metric.rego

```diff
@@ -6,10 +6,11 @@ default applicable = false
 
 default compliant = false
 
-enabled := input.webApplicationFirewall.enabled
+enabled := input.accessRestriction.enabled
```

Comment:
Is this actually working this way? When I look at webprotoger, `accessRestriction` is parent of `firewall` which is parent of `webApplicationFirewall`, i.e. `webApplicationFirewall` inherits everything from its parents but not vice versa, right?

Reply:
Same for metric l3firewall